src/SlashCommands.tsx

```diff
@@ -1022,6 +1023,26 @@ export const Commands = [
         },
         category: CommandCategories.actions,
     }),
+    new Command({
+        command: "confetti",
+        description: _td("Sends the given message with confetti"),
+        args: '<message>',
+        runFn: function(roomId, args) {
+            return success((async () => {
+                const isChatEffectsDisabled = SettingsStore.getValue('dontShowChatEffects');
+                if ((!args) || (!args && isChatEffectsDisabled)) {
```

Comment:
this simplifies to `!args` - seems like the condition may be not exactly what you want